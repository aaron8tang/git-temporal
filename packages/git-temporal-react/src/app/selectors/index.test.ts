import * as selectors from '.';
import fiveCommits from 'testHelpers/mocks/fiveCommits';
import tenCommits from 'testHelpers/mocks/tenCommits';

const testStateBase = {
  commitsByPath: {
    testPath1: {
      isFetching: false,
      commits: fiveCommits,
    },
    testPath2: {
      isFetching: false,
      commits: tenCommits,
    },
  },
};

const testState1 = Object.assign({}, testStateBase, {
  selectedPath: 'testPath1',
});

const testState2 = Object.assign({}, testStateBase, {
  selectedPath: 'testPath2',
});

// TODO: this will need more work and testing.  Right now, getFilteredCommits
// hits all of the selectors up the line

describe('selectors/getFilteredCommits', () => {
  describe('with first 5 commits from node test repo', () => {
    let commitsOut;
    beforeAll(async () => {
      commitsOut = await selectors.getFilteredCommits(testState1);
      // console.log('got commits', commitsOut);
    });
    test('it should match snapshot', () => {
      expect(commitsOut).toMatchSnapshot();
    });
    test('it should have returned 5 commits for testPath1', () => {
      expect(commitsOut.commits.length).toBe(5);
    });
  });
  describe('with first 10 commits from react test repo', () => {
    let commitsOut;
    beforeAll(async () => {
      commitsOut = await selectors.getFilteredCommits(testState2);
      // console.log('got commits', commitsOut);
    });
    test('it should match snapshot', () => {
      expect(commitsOut).toMatchSnapshot();
    });
    test('it should have returned 10 commits for testPath2', () => {
      expect(commitsOut.commits.length).toBe(10);
    });
  });

  describe('when dealing with commit containing bad author name', () => {
    let commitsOut;
    beforeAll(async () => {
      const testState = Object.assign({}, testState1);
      testState.commitsByPath.testPath1.commits[0].authorName = null;

      commitsOut = await selectors.getFilteredCommits(testState);
      // console.log('got commits', commitsOut);
    });
    test('it should match snapshot', () => {
      expect(commitsOut).toMatchSnapshot();
    });
    test('it should have returned 5 commits', () => {
      expect(commitsOut.commits.length).toBe(5);
    });
    test('first commit should not have an authorName', () => {
      expect(commitsOut.commits[0].authorName).toBe(null);
      expect(commitsOut.commits[1].authorName).not.toBe(null);
    });
  });

  describe('when asking for non existent path', () => {
    let commitsOut;
    beforeAll(async () => {
      const testState = Object.assign({}, testStateBase, {
        selectedPath: 'some/bogus/path',
      });
      commitsOut = await selectors.getFilteredCommits(testState);
    });
    test('it should match snapshot', () => {
      expect(commitsOut).toMatchSnapshot();
    });
  });
});
