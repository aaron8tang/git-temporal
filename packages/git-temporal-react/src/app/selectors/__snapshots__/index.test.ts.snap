// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`selectors/getFilteredCommits when asking for non existent path it should match snapshot 1`] = `
Object {
  "authorNames": Array [],
  "commits": Array [],
  "isFetching": true,
  "selectedPath": "some/bogus/path",
}
`;

exports[`selectors/getFilteredCommits when dealing with commit containing bad author name it should match snapshot 1`] = `
Object {
  "authorNames": Array [
    "null",
    "Daniel Bevenius",
    "Tobias Nießen",
    "Michaël Zasso",
    "Ruben Bridgewater",
  ],
  "commits": Array [
    Object {
      "authorDate": 1537541919,
      "authorName": null,
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 2,
          "linesDeleted": 649,
          "name": "README.md",
        },
      ],
      "hash": "c698a0f01d",
      "id": "c698a0f01de1fa682443510920a2dafc2440d6c8",
      "linesAdded": 2,
      "linesDeleted": 649,
      "message": "Explain why this fork exists in README.md",
      "relativeDate": "2 days ago",
    },
    Object {
      "authorDate": 1537272890,
      "authorName": "Tobias Nießen",
      "body": "Calling setAuthTag multiple times can result in hard to detect bugssince to the user, it is unclear which invocation actually affectedOpenSSL.PR-URL: https://github.com/nodejs/node/pull/22931Reviewed-By: Anna Henningsen <anna@addaleax.net>Reviewed-By: James M Snell <jasnell@gmail.com>Reviewed-By: Ujjwal Sharma <usharma1998@gmail.com>",
      "files": Array [
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "doc/api/crypto.md",
        },
        Object {
          "linesAdded": 2,
          "linesDeleted": 5,
          "name": "src/node_crypto.cc",
        },
        Object {
          "linesAdded": 26,
          "linesDeleted": 0,
          "name": "test/parallel/test-crypto-authenticated.js",
        },
      ],
      "hash": "058c5b81cd",
      "id": "058c5b81cdbabe8989a194ba5d388f4c230f4af6",
      "linesAdded": 29,
      "linesDeleted": 6,
      "message": "crypto: do not allow multiple calls to setAuthTag",
      "relativeDate": "2 days ago",
    },
    Object {
      "authorDate": 1526296167,
      "authorName": "Daniel Bevenius",
      "body": "This commit adds a --completion-bash option to node which can besourced to provide bash code completion for node options.Usage:$ node --completion-bash  > node_bash_completion$ source node_bash_completion$ node --[tab]PR-URL: https://github.com/nodejs/node/pull/20713Reviewed-By: Anna Henningsen <anna@addaleax.net>Reviewed-By: James M Snell <jasnell@gmail.com>Reviewed-By: Denys Otrishko <shishugi@gmail.com>",
      "files": Array [
        Object {
          "linesAdded": 11,
          "linesDeleted": 0,
          "name": "doc/api/cli.md",
        },
        Object {
          "linesAdded": 3,
          "linesDeleted": 0,
          "name": "doc/node.1",
        },
        Object {
          "linesAdded": 25,
          "linesDeleted": 0,
          "name": "lib/internal/bash_completion.js",
        },
        Object {
          "linesAdded": 7,
          "linesDeleted": 1,
          "name": "lib/internal/bootstrap/node.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 0,
          "name": "node.gyp",
        },
        Object {
          "linesAdded": 3,
          "linesDeleted": 0,
          "name": "src/node_options.cc",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 0,
          "name": "src/node_options.h",
        },
      ],
      "hash": "56493bf1eb",
      "id": "56493bf1ebfab3ec102fe017f30fa4f81ba6a256",
      "linesAdded": 51,
      "linesDeleted": 1,
      "message": "tools: add bash completion for node",
      "relativeDate": "2 days ago",
    },
    Object {
      "authorDate": 1537212012,
      "authorName": "Michaël Zasso",
      "body": "Replace references to the outdated \`update-v8\` tool to its replacementin \`node-core-utils\`PR-URL: https://github.com/nodejs/node/pull/22913Reviewed-By: Richard Lau <riclau@uk.ibm.com>Reviewed-By: Colin Ihrig <cjihrig@gmail.com>Reviewed-By: Weijia Wang <starkwang@126.com>Reviewed-By: Sakthipriyan Vairamani <thechargingvolcano@gmail.com>Reviewed-By: Joyee Cheung <joyeec9h3@gmail.com>Reviewed-By: Luigi Pinca <luigipinca@gmail.com>Reviewed-By: James M Snell <jasnell@gmail.com>",
      "files": Array [
        Object {
          "linesAdded": 8,
          "linesDeleted": 8,
          "name": "doc/guides/maintaining-V8.md",
        },
      ],
      "hash": "408e8ce22c",
      "id": "408e8ce22ce7436e88a9977b4dc99fc3c431fc67",
      "linesAdded": 8,
      "linesDeleted": 8,
      "message": "doc: update maintaining V8 guide",
      "relativeDate": "3 days ago",
    },
    Object {
      "authorDate": 1537269982,
      "authorName": "Ruben Bridgewater",
      "body": "Currently the documentation is not specific how fast-tracking shouldbe applied. This specifies exactly how things should be done to preventconfusion.PR-URL: https://github.com/nodejs/node/pull/22929Reviewed-By: Denys Otrishko <shishugi@gmail.com>Reviewed-By: Anna Henningsen <anna@addaleax.net>Reviewed-By: Tobias Nießen <tniessen@tnie.de>Reviewed-By: Vse Mozhet Byt <vsemozhetbyt@gmail.com>Reviewed-By: John-David Dalton <john.david.dalton@gmail.com>Reviewed-By: Matteo Collina <matteo.collina@gmail.com>Reviewed-By: Luigi Pinca <luigipinca@gmail.com>Reviewed-By: Michaël Zasso <targos@protonmail.com>Reviewed-By: Weijia Wang <starkwang@126.com>Reviewed-By: Richard Lau <riclau@uk.ibm.com>Reviewed-By: Matheus Marchini <mat@mmarchini.me>Reviewed-By: Sakthipriyan Vairamani <thechargingvolcano@gmail.com>Reviewed-By: Yuta Hiroto <hello@hiroppy.me>Reviewed-By: Rich Trott <rtrott@gmail.com>Reviewed-By: Trivikram Kamat <trivikr.dev@gmail.com>Reviewed-By: George Adams <george.adams@uk.ibm.com>",
      "files": Array [
        Object {
          "linesAdded": 6,
          "linesDeleted": 3,
          "name": "COLLABORATOR_GUIDE.md",
        },
      ],
      "hash": "54af9738b9",
      "id": "54af9738b9fad1c1e46ea9170037146324cd152e",
      "linesAdded": 6,
      "linesDeleted": 3,
      "message": "doc: specify fast-tracking",
      "relativeDate": "3 days ago",
    },
  ],
  "isFetching": false,
  "selectedPath": "testPath1",
}
`;

exports[`selectors/getFilteredCommits with first 5 commits from node test repo it should match snapshot 1`] = `
Object {
  "authorNames": Array [
    "Bee Wilkerson",
    "Daniel Bevenius",
    "Tobias Nießen",
    "Michaël Zasso",
    "Ruben Bridgewater",
  ],
  "commits": Array [
    Object {
      "authorDate": 1537541919,
      "authorName": "Bee Wilkerson",
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 2,
          "linesDeleted": 649,
          "name": "README.md",
        },
      ],
      "hash": "c698a0f01d",
      "id": "c698a0f01de1fa682443510920a2dafc2440d6c8",
      "linesAdded": 2,
      "linesDeleted": 649,
      "message": "Explain why this fork exists in README.md",
      "relativeDate": "2 days ago",
    },
    Object {
      "authorDate": 1537272890,
      "authorName": "Tobias Nießen",
      "body": "Calling setAuthTag multiple times can result in hard to detect bugssince to the user, it is unclear which invocation actually affectedOpenSSL.PR-URL: https://github.com/nodejs/node/pull/22931Reviewed-By: Anna Henningsen <anna@addaleax.net>Reviewed-By: James M Snell <jasnell@gmail.com>Reviewed-By: Ujjwal Sharma <usharma1998@gmail.com>",
      "files": Array [
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "doc/api/crypto.md",
        },
        Object {
          "linesAdded": 2,
          "linesDeleted": 5,
          "name": "src/node_crypto.cc",
        },
        Object {
          "linesAdded": 26,
          "linesDeleted": 0,
          "name": "test/parallel/test-crypto-authenticated.js",
        },
      ],
      "hash": "058c5b81cd",
      "id": "058c5b81cdbabe8989a194ba5d388f4c230f4af6",
      "linesAdded": 29,
      "linesDeleted": 6,
      "message": "crypto: do not allow multiple calls to setAuthTag",
      "relativeDate": "2 days ago",
    },
    Object {
      "authorDate": 1526296167,
      "authorName": "Daniel Bevenius",
      "body": "This commit adds a --completion-bash option to node which can besourced to provide bash code completion for node options.Usage:$ node --completion-bash  > node_bash_completion$ source node_bash_completion$ node --[tab]PR-URL: https://github.com/nodejs/node/pull/20713Reviewed-By: Anna Henningsen <anna@addaleax.net>Reviewed-By: James M Snell <jasnell@gmail.com>Reviewed-By: Denys Otrishko <shishugi@gmail.com>",
      "files": Array [
        Object {
          "linesAdded": 11,
          "linesDeleted": 0,
          "name": "doc/api/cli.md",
        },
        Object {
          "linesAdded": 3,
          "linesDeleted": 0,
          "name": "doc/node.1",
        },
        Object {
          "linesAdded": 25,
          "linesDeleted": 0,
          "name": "lib/internal/bash_completion.js",
        },
        Object {
          "linesAdded": 7,
          "linesDeleted": 1,
          "name": "lib/internal/bootstrap/node.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 0,
          "name": "node.gyp",
        },
        Object {
          "linesAdded": 3,
          "linesDeleted": 0,
          "name": "src/node_options.cc",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 0,
          "name": "src/node_options.h",
        },
      ],
      "hash": "56493bf1eb",
      "id": "56493bf1ebfab3ec102fe017f30fa4f81ba6a256",
      "linesAdded": 51,
      "linesDeleted": 1,
      "message": "tools: add bash completion for node",
      "relativeDate": "2 days ago",
    },
    Object {
      "authorDate": 1537212012,
      "authorName": "Michaël Zasso",
      "body": "Replace references to the outdated \`update-v8\` tool to its replacementin \`node-core-utils\`PR-URL: https://github.com/nodejs/node/pull/22913Reviewed-By: Richard Lau <riclau@uk.ibm.com>Reviewed-By: Colin Ihrig <cjihrig@gmail.com>Reviewed-By: Weijia Wang <starkwang@126.com>Reviewed-By: Sakthipriyan Vairamani <thechargingvolcano@gmail.com>Reviewed-By: Joyee Cheung <joyeec9h3@gmail.com>Reviewed-By: Luigi Pinca <luigipinca@gmail.com>Reviewed-By: James M Snell <jasnell@gmail.com>",
      "files": Array [
        Object {
          "linesAdded": 8,
          "linesDeleted": 8,
          "name": "doc/guides/maintaining-V8.md",
        },
      ],
      "hash": "408e8ce22c",
      "id": "408e8ce22ce7436e88a9977b4dc99fc3c431fc67",
      "linesAdded": 8,
      "linesDeleted": 8,
      "message": "doc: update maintaining V8 guide",
      "relativeDate": "3 days ago",
    },
    Object {
      "authorDate": 1537269982,
      "authorName": "Ruben Bridgewater",
      "body": "Currently the documentation is not specific how fast-tracking shouldbe applied. This specifies exactly how things should be done to preventconfusion.PR-URL: https://github.com/nodejs/node/pull/22929Reviewed-By: Denys Otrishko <shishugi@gmail.com>Reviewed-By: Anna Henningsen <anna@addaleax.net>Reviewed-By: Tobias Nießen <tniessen@tnie.de>Reviewed-By: Vse Mozhet Byt <vsemozhetbyt@gmail.com>Reviewed-By: John-David Dalton <john.david.dalton@gmail.com>Reviewed-By: Matteo Collina <matteo.collina@gmail.com>Reviewed-By: Luigi Pinca <luigipinca@gmail.com>Reviewed-By: Michaël Zasso <targos@protonmail.com>Reviewed-By: Weijia Wang <starkwang@126.com>Reviewed-By: Richard Lau <riclau@uk.ibm.com>Reviewed-By: Matheus Marchini <mat@mmarchini.me>Reviewed-By: Sakthipriyan Vairamani <thechargingvolcano@gmail.com>Reviewed-By: Yuta Hiroto <hello@hiroppy.me>Reviewed-By: Rich Trott <rtrott@gmail.com>Reviewed-By: Trivikram Kamat <trivikr.dev@gmail.com>Reviewed-By: George Adams <george.adams@uk.ibm.com>",
      "files": Array [
        Object {
          "linesAdded": 6,
          "linesDeleted": 3,
          "name": "COLLABORATOR_GUIDE.md",
        },
      ],
      "hash": "54af9738b9",
      "id": "54af9738b9fad1c1e46ea9170037146324cd152e",
      "linesAdded": 6,
      "linesDeleted": 3,
      "message": "doc: specify fast-tracking",
      "relativeDate": "3 days ago",
    },
  ],
  "isFetching": false,
  "selectedPath": "testPath1",
}
`;

exports[`selectors/getFilteredCommits with first 10 commits from react test repo it should match snapshot 1`] = `
Object {
  "authorNames": Array [
    "Dan Abramov",
    "Brian Vaughn",
    "Bee Wilkerson",
    "Dan",
    "Maksim Markelov",
    "Heaven",
  ],
  "commits": Array [
    Object {
      "authorDate": 1537542150,
      "authorName": "Bee Wilkerson",
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 2,
          "linesDeleted": 74,
          "name": "README.md",
        },
      ],
      "hash": "abbf59e3a",
      "id": "abbf59e3af03cc872a52a8bb3a1dcfeb1b60d9bb",
      "linesAdded": 2,
      "linesDeleted": 74,
      "message": "Explain why this fork exists in README.md",
      "relativeDate": "2 days ago",
    },
    Object {
      "authorDate": 1537379312,
      "authorName": "Maksim Markelov",
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/scheduler/src/TracingSubscriptions.js",
        },
      ],
      "hash": "e1a067dea",
      "id": "e1a067dea0ffcacd1f664f30cd14463b00f52fa7",
      "linesAdded": 1,
      "linesDeleted": 1,
      "message": "Fix circular dependency in TracingSubscriptions (#13689)",
      "relativeDate": "4 days ago",
    },
    Object {
      "authorDate": 1537359272,
      "authorName": "Heaven",
      "body": "* fix comment typo <br> <br>* Update Scheduler.js <br>",
      "files": Array [
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/scheduler/src/Scheduler.js",
        },
      ],
      "hash": "518812eeb",
      "id": "518812eeb82a524dd264be79ca0573ce39cde945",
      "linesAdded": 1,
      "linesDeleted": 1,
      "message": "Clarify comment (#13684)",
      "relativeDate": "4 days ago",
    },
    Object {
      "authorDate": 1537317391,
      "authorName": "Dan",
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 0,
          "linesDeleted": 42,
          "name": "scripts/rollup/results.json",
        },
      ],
      "hash": "eeb817785",
      "id": "eeb817785c771362416fd87ea7d2a1a32dde9842",
      "linesAdded": 0,
      "linesDeleted": 42,
      "message": "Remove some old files from stats",
      "relativeDate": "5 days ago",
    },
    Object {
      "authorDate": 1537316788,
      "authorName": "Dan Abramov",
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 5,
          "linesDeleted": 4,
          "name": "fixtures/{schedule => scheduler}/index.html",
        },
        Object {
          "linesAdded": 2,
          "linesDeleted": 2,
          "name": "fixtures/tracing/index.html",
        },
        Object {
          "linesAdded": 26,
          "linesDeleted": 26,
          "name": "fixtures/tracing/script.js",
        },
        Object {
          "linesAdded": 5,
          "linesDeleted": 2,
          "name": "fixtures/unstable-async/suspense/src/components/App.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "fixtures/unstable-async/suspense/src/index.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "fixtures/unstable-async/time-slicing/README.md",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "fixtures/unstable-async/time-slicing/src/index.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/react-art/package.json",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/react-art/src/ReactARTHostConfig.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/react-dom/package.json",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/react-dom/src/client/ReactDOMHostConfig.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/react-native-renderer/package.json",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/react-reconciler/package.json",
        },
        Object {
          "linesAdded": 2,
          "linesDeleted": 2,
          "name": "packages/react-reconciler/src/ReactFiberRoot.js",
        },
        Object {
          "linesAdded": 2,
          "linesDeleted": 2,
          "name": "packages/react-reconciler/src/ReactFiberScheduler.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/react-reconciler/src/__tests__/ReactTracing-test.internal.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/react-test-renderer/package.json",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/react/package.json",
        },
        Object {
          "linesAdded": 4,
          "linesDeleted": 4,
          "name": "packages/react/src/ReactSharedInternals.js",
        },
        Object {
          "linesAdded": 5,
          "linesDeleted": 5,
          "name": "packages/react/src/__tests__/ReactProfiler-test.internal.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/react/src/__tests__/ReactProfilerDevToolsIntegration-test.internal.js",
        },
        Object {
          "linesAdded": 0,
          "linesDeleted": 7,
          "name": "packages/schedule/npm/index.js",
        },
        Object {
          "linesAdded": 0,
          "linesDeleted": 7,
          "name": "packages/schedule/npm/tracing-profiling.js",
        },
        Object {
          "linesAdded": 0,
          "linesDeleted": 7,
          "name": "packages/schedule/npm/tracing.js",
        },
        Object {
          "linesAdded": 2,
          "linesDeleted": 2,
          "name": "packages/{schedule => scheduler}/README.md",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/{schedule => scheduler}/index.js",
        },
        Object {
          "linesAdded": 7,
          "linesDeleted": 0,
          "name": "packages/scheduler/npm/index.js",
        },
        Object {
          "linesAdded": 7,
          "linesDeleted": 0,
          "name": "packages/scheduler/npm/tracing-profiling.js",
        },
        Object {
          "linesAdded": 7,
          "linesDeleted": 0,
          "name": "packages/scheduler/npm/tracing.js",
        },
        Object {
          "linesAdded": 8,
          "linesDeleted": 8,
          "name": "packages/{schedule/npm/umd/schedule-tracing.profiling.min.js => scheduler/npm/umd/scheduler-tracing.development.js}",
        },
        Object {
          "linesAdded": 8,
          "linesDeleted": 8,
          "name": "packages/{schedule/npm/umd/schedule-tracing.production.min.js => scheduler/npm/umd/scheduler-tracing.production.min.js}",
        },
        Object {
          "linesAdded": 8,
          "linesDeleted": 8,
          "name": "packages/{schedule/npm/umd/schedule-tracing.development.js => scheduler/npm/umd/scheduler-tracing.profiling.min.js}",
        },
        Object {
          "linesAdded": 4,
          "linesDeleted": 4,
          "name": "packages/{schedule/npm/umd/schedule.production.min.js => scheduler/npm/umd/scheduler.development.js}",
        },
        Object {
          "linesAdded": 4,
          "linesDeleted": 4,
          "name": "packages/{schedule/npm/umd/schedule.profiling.min.js => scheduler/npm/umd/scheduler.production.min.js}",
        },
        Object {
          "linesAdded": 4,
          "linesDeleted": 4,
          "name": "packages/{schedule/npm/umd/schedule.development.js => scheduler/npm/umd/scheduler.profiling.min.js}",
        },
        Object {
          "linesAdded": 2,
          "linesDeleted": 2,
          "name": "packages/{schedule => scheduler}/package.json",
        },
        Object {
          "linesAdded": 0,
          "linesDeleted": 0,
          "name": "packages/{schedule/src/Schedule.js => scheduler/src/Scheduler.js}",
        },
        Object {
          "linesAdded": 0,
          "linesDeleted": 0,
          "name": "packages/{schedule => scheduler}/src/Tracing.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/{schedule => scheduler}/src/TracingSubscriptions.js",
        },
        Object {
          "linesAdded": 4,
          "linesDeleted": 4,
          "name": "packages/{schedule/src/__tests__/Schedule-test.internal.js => scheduler/src/__tests__/Scheduler-test.internal.js}",
        },
        Object {
          "linesAdded": 19,
          "linesDeleted": 19,
          "name": "packages/{schedule/src/__tests__/ScheduleDOM-test.js => scheduler/src/__tests__/SchedulerDOM-test.js}",
        },
        Object {
          "linesAdded": 8,
          "linesDeleted": 8,
          "name": "packages/{schedule/src/__tests__/ScheduleUMDBundle-test.internal.js => scheduler/src/__tests__/SchedulerUMDBundle-test.internal.js}",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/{schedule => scheduler}/src/__tests__/Tracing-test.internal.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/{schedule => scheduler}/src/__tests__/Tracing-test.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/{schedule => scheduler}/src/__tests__/TracingSubscriptions-test.internal.js",
        },
        Object {
          "linesAdded": 0,
          "linesDeleted": 0,
          "name": "packages/{schedule => scheduler}/tracing.js",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/shared/forks/{Schedule.umd.js => Scheduler.umd.js}",
        },
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "packages/shared/forks/{ScheduleTracing.umd.js => SchedulerTracing.umd.js}",
        },
        Object {
          "linesAdded": 6,
          "linesDeleted": 6,
          "name": "scripts/rollup/bundles.js",
        },
        Object {
          "linesAdded": 4,
          "linesDeleted": 4,
          "name": "scripts/rollup/forks.js",
        },
        Object {
          "linesAdded": 4,
          "linesDeleted": 4,
          "name": "scripts/rollup/modules.js",
        },
        Object {
          "linesAdded": 36,
          "linesDeleted": 36,
          "name": "scripts/rollup/results.json",
        },
      ],
      "hash": "7ea3ca1d1",
      "id": "7ea3ca1d13b1b609678fa1369f8a1020c3ecb976",
      "linesAdded": 213,
      "linesDeleted": 209,
      "message": "Rename schedule to scheduler (#13683)",
      "relativeDate": "5 days ago",
    },
    Object {
      "authorDate": 1537299931,
      "authorName": "Brian Vaughn",
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 2,
          "linesDeleted": 1,
          "name": "CHANGELOG.md",
        },
      ],
      "hash": "9b7081664",
      "id": "9b70816642c68672da97e911d6d351fd3c596474",
      "linesAdded": 2,
      "linesDeleted": 1,
      "message": "Added another bullet to the CHANGELOG",
      "relativeDate": "5 days ago",
    },
    Object {
      "authorDate": 1537299674,
      "authorName": "Brian Vaughn",
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 1,
          "linesDeleted": 1,
          "name": "CHANGELOG.md",
        },
      ],
      "hash": "db9d51b65",
      "id": "db9d51b65caeec0cfd2b8e35df59bdaba3eaa31b",
      "linesAdded": 1,
      "linesDeleted": 1,
      "message": "Rename 'Schedule' header -> 'Schedule (Experimental)'",
      "relativeDate": "5 days ago",
    },
    Object {
      "authorDate": 1537299572,
      "authorName": "Brian Vaughn",
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 12,
          "linesDeleted": 0,
          "name": "CHANGELOG.md",
        },
      ],
      "hash": "0823f845c",
      "id": "0823f845cf38141866e7e5164c80d75b4985df04",
      "linesAdded": 12,
      "linesDeleted": 0,
      "message": "16.5.2 CHANGELOG",
      "relativeDate": "5 days ago",
    },
    Object {
      "authorDate": 1537295450,
      "authorName": "Brian Vaughn",
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 125,
          "linesDeleted": 69,
          "name": "scripts/rollup/results.json",
        },
      ],
      "hash": "bec2ddaf1",
      "id": "bec2ddaf157497ddca85e823e180536d8154a52f",
      "linesAdded": 125,
      "linesDeleted": 69,
      "message": "Update bundle sizes for 16.5.2 release",
      "relativeDate": "5 days ago",
    },
    Object {
      "authorDate": 1537295450,
      "authorName": "Brian Vaughn",
      "body": "",
      "files": Array [
        Object {
          "linesAdded": 2,
          "linesDeleted": 1,
          "name": "scripts/error-codes/codes.json",
        },
      ],
      "hash": "789e714bd",
      "id": "789e714bd738b0dd566f3cff33932b9586751fc5",
      "linesAdded": 2,
      "linesDeleted": 1,
      "message": "Update error codes for 16.5.2 release",
      "relativeDate": "5 days ago",
    },
  ],
  "isFetching": false,
  "selectedPath": "testPath2",
}
`;
